### PR TITLE
Update UserMenu.cshtml

### PR DESCRIPTION
**Pull request by MikeAlhayek**

### CODE REVIEW

File: src/OrchardCoreContrib.Gravatar/Views/UserMenu.cshtml [hunk 1]
```diff
@@ -32,6 +29,9 @@
                 <li><a class="dropdown-item" asp-area="OrchardCore.Users" asp-action="Display" asp-controller="Admin" asp-route-id="@userId" asp-route-returnUrl="@FullRequestPath"><span><i class="far fa-address-card" aria-hidden="true"></i></span>&nbsp;<span> @T["Profile"]</span></a></li>
             }
             <li><a class="dropdown-item" asp-area="OrchardCore.Users" asp-controller="Account" asp-action="ChangePassword" asp-route-returnUrl="@FullRequestPath"><span><i class="fas fa-key" aria-hidden="true"></i></span>&nbsp;<span> @T["Change password"]</span></a></li>
+            
+            @await RenderSectionAsync("UserMenuItems", required: false)
```
Comment (hishamco) [inline src/OrchardCoreContrib.Gravatar/Views/UserMenu.cshtml:33]: Why do we need this?

Comment (MikeAlhayek) [reply src/OrchardCoreContrib.Gravatar/Views/UserMenu.cshtml:33]: It's something new we added to allow others to inject items in this menu. We should at some point submit a PR to remove most of these menu items from the view and use filters to inject them into the view

Comment (hishamco) [reply src/OrchardCoreContrib.Gravatar/Views/UserMenu.cshtml:33]: Happy to hear that, I will have a look and merge your PR ASAP

Comment (hishamco) [reply src/OrchardCoreContrib.Gravatar/Views/UserMenu.cshtml:33]: @MikeAlhayek I presume this should be in the Users module, right? after that we can inject the menu items from other modules

Comment (MikeAlhayek) [reply src/OrchardCoreContrib.Gravatar/Views/UserMenu.cshtml:33]: It should only be in UserMenu view which will optionally allow others to inject into this zone

File: src/OrchardCoreContrib.Gravatar/Views/UserMenu.cshtml [hunk 2]
```diff
@@ -20,10 +20,7 @@
         <ul class="dropdown-menu dropdown-menu-end" aria-labelledby="navbarDropdown">
             <li><h6 class="dropdown-header">@T["Signed in as"]<span class="ms-1 font-weight-bold">@User.Identity.Name</span></h6></li>
             <li><hr class="dropdown-divider" /></li>
-            @*
-            TODO: Use CommonPermissions.EditOwnUser once 1.6.0 is released
-            *@
-            @if (await AuthorizationService.AuthorizeAsync(User, CommonPermissions.ManageUsers))
+            @if (await AuthorizationService.AuthorizeAsync(User, CommonPermissions.EditOwnUser))
```
Comment (hishamco) [inline src/OrchardCoreContrib.Gravatar/Views/UserMenu.cshtml:23]: The build failed, we need to upgrade to `1.6.0`, I will update in another PR then I will merge this

**Comment by hishamco** [comment]
Could you please rebase, so I can merge this?

**Comment by MikeAlhayek** [comment]
done

**Comment by hishamco** [comment]
Please do it one more time, there was a build issue, that has been fixed and merged

**Comment by hishamco** [comment]
Thanks a lot

**Pull request merged**